docs/contributor/resources/01-kyma.md

```diff
@@ -81,6 +81,18 @@ spec:
 > **CAUTION:**
 > Module referencing using NamespacedName and FQDN (Fully Qualified Domain Name) has been deprecated.
 
+### **.spec.modules[].managed**
+
+The **managed** field determines whether Lifecycle Manager manages the module or not. By default it is set to `true`. By setting it to `false`, the module is opted out of management by Lifecycle Manager. This means that the module and all its related resources remain in the remote cluster in their current state at the time of unmanaging the module, but Lifecycle Manager does NOT reconcile those any longer.
+
+Once a module has been successfully unmanaged, the module's status in **.status.modules[].state** shows `Unmanaged`. Once the `Unmanaged` state shows, the module's entry can also be deleted entirely from **.spec.modules[]**.
+
+If the **.spec.modules[].managed** field is set to `true` again, Lifecycle Manager picks up the management of the module again. Existing module resources in the remote cluster are left in tact but may be overwritten if the desired state has changed in the meantime, e.g., if the version within the used channel has progressed in the meantime.
+
+
+> **CAUTION:**
+> When switching between values of **.spec.modules[].managed**, the user MUST wait for the new state to be reflected in **.status.modules[].state** before removing the module's entry from **.status.modules[]**. If the entry is removed before the current state is reflected properly in **.status.modules[].state**, it may lead to unpredictable behavior that is hard to be recovered from.
```

Comment:
```suggestion
> When you switch values of **.spec.modules[].managed**, you MUST wait for the new state to be reflected in **.status.modules[].state** before you remove the module's entry from **.status.modules[]**. If the entry is removed before the current state is reflected properly in **.status.modules[].state**, it may lead to unpredictable behavior that is hard to recover from.
```